tools/testing/e2e/test_utils.c

```diff
@@ -411,6 +417,10 @@ void iree_test_utils_get_min_max_for_element_type(
       *min = 0;
       *max = +16;
       break;
+    case IREE_HAL_ELEMENT_TYPE_FLOAT_8_E8M0_FNU:
+      *min = 1;
```

Comment:
... these are really small scales. You'll want to orbit around 127 +- a few?

Reply:
Or are these meant to be read as float values? In which case you might want 1/2, 1/4, and a few other nearby fractions?